packages/vscode-extension/package.json

```diff
@@ -108,80 +92,82 @@
+        "contents": "To start Teams app development experience, create a project or explore our samples. For more information, visit [Get Started](command:fx-extension.openWelcome?%5B%22SideBar%22%5D), [Tutorials](command:fx-extension.selectTutorials?%5B%22SideBar%22%5D) or [Documentation](command:fx-extension.openDocument?%5B%22SideBar%22%5D).\n[Create a new Teams app](command:fx-extension.create?%5B%22SideBar%22%5D)\n[View samples](command:fx-extension.openSamples?%5B%22SideBar%22%5D)\nYou can also open an existing Teams app.\n[Open folder](command:vscode.openFolder)\nYou need to upgrade the project to the new configuration files to use the latest features. The upgrade process will not change your custom code and create the backup files in case revert is needed.\nNotice this upgrade is a must-have to continue to use current version Teams Toolkit. If you are not ready to upgrade and want to continue to use the old version Teams Toolkit, please find Teams Toolkit in Extension and install the version <=2.10.0.\n[Upgrade project](command:fx-extension.checkProjectUpgrade)",
+        "enablement": "fx-extension.initialized"
       }
     ],
     "views": {
       "teamsfx": [
         {
           "id": "teamsfx-accounts",
           "name": "Accounts",
-          "when": "fx-extension.sidebarWelcome.treeview"
+          "when": "fx-extension.isTeamsFx"
         },
         {
           "id": "teamsfx-environment",
           "name": "Environment",
-          "when": "fx-extension.sidebarWelcome.treeview"
+          "when": "fx-extension.isTeamsFx"
         },
         {
           "id": "teamsfx-development",
           "name": "Development",
-          "when": "fx-extension.sidebarWelcome.treeview && !fx-extension.customizedTreeview"
+          "when": "fx-extension.isTeamsFx"
         },
         {
           "type": "webview",
           "id": "teamsfx-development-webview",
           "name": "Development",
-          "when": "fx-extension.sidebarWelcome.treeview && fx-extension.customizedTreeview"
+          "when": "fx-extension.isTeamsFx && fx-extension.customizedTreeview"
         },
         {
           "id": "teamsfx-deployment",
           "name": "Deployment",
-          "when": "fx-extension.sidebarWelcome.treeview && !fx-extension.customizedTreeview"
+          "when": "fx-extension.isTeamsFx"
         },
         {
           "type": "webview",
           "id": "teamsfx-deployment-webview",
           "name": "Deployment",
-          "when": "fx-extension.sidebarWelcome.treeview && fx-extension.customizedTreeview"
+          "when": "fx-extension.isTeamsFx && fx-extension.customizedTreeview"
```

Comment:
Remove customizedTreeView here to align with other commands?

Reply:
It is kept for the legacy webview as treeview, can we remove the whole item?

Reply:
OK, I think we can keep for now.